src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -360,13 +361,16 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     * @return TaskStatus with final status of the task.
     */
   private def processOutputResult(processReturnCode: Int, outputsExpressions: Seq[(String, OutputStmtEval)]): TaskFinalStatus = {
-    if (outputsExpressions.filter(_._2.rhs.isFailure).size > 0) {
+    if (outputsExpressions.exists(_._2.rhs.isFailure)) {
```

Comment:
Did you consider a case class for the (String, OutputStmtEval) tuple?

I have a minor hatred of _._2s